### ISSUE TITLE
Reappearing Folder in AppData

### ISSUE THREAD

**Issue by please55** [HearthSim/Hearthstone-Deck-Tracker#3803]
Got portable version. Extracted zip to C:\
Ran HDT and disabled both 'Save config & data in AppData' options.
Set Data Path to same location in C:\ as unzipped folder.
Now even though I deleted the original 'HearthstoneDeckTracker' folder in %AppData/Roaming, it keeps reappearing after running HDT.  It contains Plugins folder that is empty (non installed atm, however the 'Plugin Folder' in options redirects there) and Last Games.xml.

I was hoping to have the whole program contained to one folder; is that possible? Not sure if this is bug per se or intended behaviour.  If so, a curious why and is there any way to change it?

**Comment by riQQ** [comment]
Most likely a bug.

**Comment by ceceita** [comment]
yeah, it's becouse at some point i dont know why they decided to not load plugins from install folder , infact if the install plugin folder the one you use with portable version they put this file
 "READ THIS.txt"

> PLUGINS INSTALLED TO THIS DIRECTORY WILL BE REMOVED!
> 
> Please install your new plugins to '%AppData%/HearthstoneDeckTracker'.
> 'options > tracker > plugins > plugins folder' will open that directory for you.

so i think that force the creation of the new folder.

So the behaviour is intended , but is bad for who want to use it as portable whitout placing new files on pc.

**Comment by beheh** [comment]
Closing, as the portable version is no longer supported.
